pkg/universe.dagger.io/x/robvanderlee@jvszonwering.nl/composer/composer.cue

```diff
@@ -0,0 +1,77 @@
+// Composer is a package manager for PHP applications
+package composer
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/bash"
+	"universe.dagger.io/docker"
+)
+
+// Run a composer install command
+#Run: {
+	// Custom name for this command.
+	// Assign an app-specific name if there are multiple apps in the same plan.
+	name: string | *""
+
+	// App source code
+	source: dagger.#FS
+
+	// Optional arguments for the script
+	args: [...string] | *[]
```

Comment:
I just checked, and it seems that  `args: [...string]` is equivalent to `args: [...string] | *[]` in Cue. Am I missing a use-case ? https://cuelang.org/play/?id=eGtZM90QbWz#cue@export@cue